tests/backup/backup_resiliency_test.go

```diff
@@ -1312,10 +1312,12 @@ var _ = Describe("{RebootNodesWhenBackupsAreInProgress}", func() {
 		log.Infof("Deleting the deployed applications on application cluster")
 		opts := make(map[string]bool)
 		opts[SkipClusterScopedObjects] = true
-		ValidateAndDestroy(contexts, opts)
+		DestroyApps(contexts, opts)
```

Comment:
Can we have all testcases replaced with DestroyApps , I think this ValidateAndDestroy takes 2-3 mins of execution, if we are replacing it in all testcases it would save significant time from the overall execution time of nightly run.

Reply:
@ak-px it would be a good improvement  to do but not in this PR. This PR is mostly to do a quick fix in this testcase only.

Reply:
Fine, It can be taken in another PR. Thanks.